graders/python/python_autograder/pl_execute.py

```diff
@@ -132,10 +129,12 @@ def execute_code(
     # ref_code contains the correct answers
 
     if include_plt:
-        for i, j in ref_code.items():
-            if isinstance(j, ModuleType):
-                if j.__dict__["__name__"] == "matplotlib.pyplot":
-                    j.close("all")
+        for _, j in ref_code.items():
```

Comment:
Could we just do this instead? And is there a better name than `j`?

```suggestion
        for j in ref_code.values():
```

Reply:
There is probably a better variable name than `j`, I am not familiar enough with the code to propose a good change. This code has this issue throughout (using i, j everywhere).

Reply:
Switched to values in cb0dd63df

Reply:
I personally don't want to handle the renames in this PR, and would rather focus on just getting the linting rules up, if that is OK.